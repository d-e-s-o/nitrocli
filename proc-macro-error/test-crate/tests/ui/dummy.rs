extern crate test_crate;
use test_crate::make_fn;

enum NeedDefault {
    A,
    B
}

make_fn!(need_default);

fn main() {
    let _ = NeedDefault::default();
}



