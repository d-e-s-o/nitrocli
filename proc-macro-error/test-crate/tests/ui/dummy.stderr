error: set_dummy test
 --> $DIR/dummy.rs:9:10
  |
9 | make_fn!(need_default);
  |          ^^^^^^^^^^^^
